src/encoded/static/components/matrix_deeply_profiled_uniform_batch.js

```diff
@@ -67,17 +68,17 @@ const sortByDescending = (a, b) => {
 const matrixDescription = 'Cell line samples that were deeply profiled using a set of diverse biochemical approaches, listed with the corresponding batch identifiers.';
 
 /** Determines if user is on All-page or not  */
-const isAllDeeplyMatrix = (url) => url.includes('deeply-profiled-matrix');
+const isAllDeeplyMatrix = (pageUrl) => pageUrl.includes('deeply-profiled-matrix');
 
 /** Switch page */
 const switchDeeplyProfilePageType = (isAll, pageUrl, navigate) => {
-    const query = new QueryString(pageUrl);
-    query.deleteKeyValue(internalTagKey);
-    const queryStringKeyValues = query.format().split('/?')[1];
-    const deeplyProfileType = isAll ? 'deeply-profiled-matrix' : 'deeply-profiled-uniform-batch-matrix';
-    const internalTagQueryString = deeplyProfileType === 'deeply-profiled-uniform-batch-matrix' ? `${internalTagKey}=Deeply Profiled` : '';
+    const link = url.parse(pageUrl, true);
```

Comment:
Well, not what I expected. The point I was making about using url.parse was so you can pass `url.parse().search` to `new QueryString()`. Now there’s not much point to using `url.parse()` since you’re manipulating the query string manually without QueryString anyway.